src/MediaRental/Controller/StoreController.java

```diff
@@ -11,28 +11,26 @@
  */
 public class StoreController
 {
+	private Store store;
 	static DatabaseSupport db = new DatabaseSupport();
-	
+
 	/**
-	 * @param name the name of the customer to add
+	 * @param name    the name of the customer to add
 	 * @param address the address of the customer to add
 	 * @return true on success, false otherwise
 	 */
-	public static boolean AddCustomer(String name, String address) {
-		DatabaseSupport db = new DatabaseSupport();
-		Customer customer = new Customer(name, address);
-		customer.setId(db.addCustomer(customer));
-		return db.addCustomer(customer) != 0;
-
+	public boolean addCustomer(String name, String address, int id)
```

Comment:
This should not take in an id. The id should be set by the database automatically